tests/models/test_models.py

```diff
@@ -1101,4 +1179,4 @@ def test_user_get_data_access_report(clean_db, monkeypatch):
         if user == admin_user:
             assert set(["*"]) == set(user_df.permissions)
         else:
-            assert set(["wes,cytof"]) == set(user_df.permissions)
+            assert set(user_df.permissions) - set(["wes,cytof", "cytof,wes"]) == set()
```

Comment:
This is an unrelated fix for a test that I found fails non-deterministically.

Reply:
there's also https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertSetEqual

Reply:
Good to know! We never expect set equality here, though, since `users_df` only has one row.

Reply:
nit pick but something like
```suggestion
            assert set(user_df.permissions) < set(["wes,cytof", "cytof,wes"])
```